mods/ts/sequences/structures.yaml

```diff
@@ -524,6 +524,84 @@ gawall:
 		Offset: 0, 0
 		UseTilesetCode: false
 
+gagate_a:
+	Defaults:
+		Offset: -24, -24
+		UseTilesetCode: true
+	idle:
+		Length: 10
```

Comment:
21 should be ok I think, [0-9] is idle, [10-19] is damaged-idle, 20 is dead, [21-30] should then be idle shadow etc